src/mbed_tools/project/project.py

```diff
@@ -79,4 +81,4 @@ def get_known_libs(path: pathlib.Path) -> Dict[str, Any]:
         detected.
```

Comment:
The `Returns` documentation needs updating, as it says a dictionary is returned when we now return a list.

Reply:
Documented.